activesupport/lib/active_support/syntax_error_proxy.rb

```diff
@@ -18,6 +18,9 @@ def spot(_)
 
       def label
       end
+
+      def base_label
```

Comment:
Is this why `.to_s` is called on `label/base_label`? If so maybe we should return empty string or something.

Reply:
I don't think it's possible for there to be a nil key (any more than a blank string key) in the hash we're using to find built_templates. I was being cautious, but given that we require Ruby >= 3.2, I'm guessing anything strange from earlier versions has been resolved.